internal/legacy/helper/schema/schema.go

```diff
@@ -252,6 +252,8 @@ type Schema struct {
 	// secret fields. Future versions of OpenTofu may encrypt these
 	// values.
 	Sensitive bool
+
+	WriteOnly bool
```

Comment:
Given that this is in the legacy package, do we need to add WriteOnly here?

Reply:
Definitely not needed. Removed in dd64b8c13c3e5ca5ae1ddfd4b3abea9cb864417f